code/.clang-format

```diff
@@ -0,0 +1,6 @@
+BasedOnStyle: WebKit
+IndentWidth: 2
+InsertNewlineAtEOF: True
+PointerAlignment: Middle
```

Comment:
I personally prefer left here, but right is also common in some code bases. So yeah, maybe middle is the right way to mess with both types of people :)

Reply:
I would just stick the the default (also for the number of spaces). We can have the usual war on tastes, but the easiest is clearly to have no change at all to the default, so drop all these extra lines after WebKit.

Reply:
OK for me to drop all the lines, but we need the new line at the end of the file. Otherwise, the pre-commit CI will add a commit to every PR where this is not the case. I will apply the changes and merge afterwards.